test/ng/compileSpec.js

```diff
@@ -3137,6 +3137,54 @@ describe('$compile', function() {
     });
 
 
+    it('should support custom start interpolation symbol, even when `endSymbol` doesn\'t change',
```

Comment:
I meant to change it as @Narretz suggested upon merge, but forgot to amend the commit before pushing :disappointed: